.github/workflows/CI.yml

```
@@ -44,9 +44,10 @@ jobs:
           version: ${{ matrix.version }}
           arch: ${{ matrix.arch }}
       - uses: julia-actions/cache@v2
-      - uses: julia-actions/julia-downgrade-compat@v1
+      - uses: julia-actions/julia-downgrade-compat@v2
         with:
           skip: LinearAlgebra,Markdown,Printf,Random,Statistics,Test
+          julia_version: ${{ matrix.version }}
```

I think we need to provide the exact version we used, which is set as an output variable by setup-julia here: https://github.com/julia-actions/setup-julia/blob/75f97ff30bbe2fe30013a0582312bd7d2e8ffb80/action.yml#L25C3-L26
```suggestion
          julia_version: ${{ matrix.julia-version }}
```

---

Good catch. I assumed it worked based on the README but now I realize that the configuration example in https://github.com/julia-actions/julia-downgrade-compat?tab=readme-ov-file#example doesn't even set `julia_version`...